pkg/operator/ceph/cluster/controller.go

```diff
@@ -323,6 +342,10 @@ func (c *ClusterController) configureExternalCephCluster(namespace, name string,
 }
 
 func (c *ClusterController) configureLocalCephCluster(namespace, name string, cluster *cluster, clusterObj *cephv1.CephCluster) error {
+	if err := c.commonCephClusterValidation(cluster); err != nil {
+		return fmt.Errorf("failed to validate external cluster spec. %+v", err)
```

Comment:
Wrong error message, it needs to be generic since it' not specific to local or external.